docs/quick_start.md

```diff
@@ -3,7 +3,54 @@ In this quick start guide, you'll get up and running with SQLMesh's scaffold gen
 
 Before beginning, ensure that you meet all the [prerequisites](prerequisites.md) for using SQLMesh.
 
-## 1. Create a SQLMesh project
+## Project structure
+
+This project demonstrates key SQLMesh features by walking through the SQLMesh workflow on a simple data pipeline. This section describes the project structure and the SQLMesh concepts you will encounter as you work through it.
+
+The project structure is a three-model pipeline with a CSV file as the only data source: 
+
+```
+┌─────────────┐
+│seed_data.csv│
+└────────────┬┘
+             │
+            ┌▼─────────────┐
+            │seed_model.sql│
+            └─────────────┬┘
+                          │
+                         ┌▼────────────────────┐
+                         │incremental_model.sql│
+                         └────────────────────┬┘
+                                              │
+                                             ┌▼─────────────┐
+                                             │full_model.sql│
+                                             └──────────────┘
+```
+
+Although the pipeline is simple, the project touches on all the primary concepts needed to use SQLMesh productively. 
+
+### Plans
+
+SQLMesh's key actions are creating and applying *plans* to *environments*. 
+
+A [SQLMesh environment](./concepts/environments.md) is an isolated namespace containing models and the data they generated. The most important environment is `prod` ("production"), which consists of the databases behind the applications your business uses to operate each day. Environments other than `prod` provide a place where you can test and preview changes to model code before they go live and affect business operations. 
+
+A [SQLMesh plan](./concepts/plans.md) contains a comparison of one environment to another and the set of changes needed to bring them into alignment. For example, if a new SQL model was added, tested, and run in the `dev` environment, it would need to be added and run in the `prod` environment to bring them into alignment. SQLMesh identifies all such changes and classifies them as either breaking and non-breaking. 
```

Comment:
```suggestion
A [SQLMesh plan](./concepts/plans.md) contains a comparison of one environment to another and the set of changes needed to bring them into alignment. For example, if a new SQL model was added, tested, and run in the `dev` environment, it would need to be added and run in the `prod` environment to bring them into alignment. SQLMesh identifies all such changes and classifies them as either breaking or non-breaking. 
```